config/_default/params.en.yaml

```
@@ -1,7 +1,7 @@
 site_name: "Datadog Infrastructure and Application Monitoring"
 meta_title: Getting Started with Datadog
 meta_description: Datadog, the leading service for cloud-scale monitoring.
-disclaimer: "This page is not yet available in English, we are working on its translation. <br> May you have any questions or feedback about our current translation project, <a href=\"https://docs.datadoghq.com/fr/help/\">feel free to reach out to us!</a>"
+disclaimer: "This page is not yet available in English, we are working on its translation. <br> Should you have any questions or feedback about our current translation project, <a href=\"https://docs.datadoghq.com/fr/help/\">feel free to reach out to us!</a>"
```

```suggestion
disclaimer: "This page is not yet available in English. We are working on its translation. <br> If you have any questions or feedback about our current translation project, <a href=\"https://docs.datadoghq.com/fr/help/\">feel free to reach out to us!</a>"
```

"Should" is a more ambiguous word; I think "If" is the most clear option.

---

Even better. Thanks!